sympy/printing/tests/test_ccode.py

```diff
@@ -119,6 +118,54 @@ def test_ccode_boolean():
     assert ccode((x & y) | z) == "z || x && y"
     assert ccode((x | y) & z) == "z && (x || y)"
 
+def test_ccode_Eq():
+    from sympy import Eq
+    from sympy import Le
+    x,y,z = symbols('x,y,z')
+    expr = Eq(x, Le(y,z))
+    assert ccode(expr) == "x == y <= z"
+
+def test_ccode_Ne():
+    from sympy import Ne
+    x,y = symbols('x,y')
+    expr = Ne(x,y)
+    assert ccode(expr) == "x != y"
+
+def test_ccode_Le():
```

Comment:
two blank lines between `def` blocks (but don't go changing the whole file, just around your new bits)